### PR TITLE
Feat/academy decentralized identity

### PR DESCRIPTION
**Pull request by MdechampG**
Add the page "Decentralized identity" in the Academy.

Still to do : 
- Add the illustrations
- Add links towards pages that don't exist yet

## Summary by CodeRabbit

- **Documentation**
  - Introduced a new section on Decentralized Identity, covering its roles, functionalities, and applications.
  - Updated the Academy Overview with renamed sections and added content related to decentralized identity.

### CODE REVIEW

File: docs/academy/decentralized-identity.mdx [hunk 1]
```diff
@@ -0,0 +1,198 @@
+---
+sidebar_class_name: hidden
+---
+
+# Decentralized identity
+
+<i>Reading time: {readingTime} min</i>
```
[the remark this answers is not in the document]
Comment (MdechampG) [reply docs/academy/decentralized-identity.mdx:7]: It calls a function that calculates it.

File: docs/academy/decentralized-identity.mdx [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+
+An Issuer is an entity that issues trustworthy data in the form of **Verifiable Credentials**. Issuers can be institutional organizations (government, bank, insurance company, etc.) or any other person or organization (a friend, a local business, a DAO, etc.). The only requirement for an issuer is its ability to certify a fact or characteristic relating to another person. 
+
+### Holders
+
+A Holder can be a person, organization, product, or object possessing a set of attributes certified by an Issuer. This holder can store and manage these attributes in the form of Verifiable Credentials and decide with whom to share this information. 
+
+### Verifiers
+
+A Verifier is an entity empowered to authenticate the legitimacy and veracity of a Verifiable Credential through a Verifiable Presentation submitted to it. The verifier can confirm that the information presented originates from the correct and authorized Issuer and that the Verifiable Credential has not been altered in any way. Generally, the verifier can also check that the Verifiable Credential has not expired or been revoked.
```
[the remark this answers is not in the document]
Comment (MdechampG) [reply docs/academy/decentralized-identity.mdx:31]: The example is described in the next part "Example : apply for a job"

File: docs/academy/decentralized-identity.mdx [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+
+**ILLUSTRATION**
```
[the remark this answers is not in the document]
Comment (MdechampG) [reply docs/academy/decentralized-identity.mdx:33]: I will add it later.

File: docs/academy/decentralized-identity.mdx [hunk 4]
```diff
@@ -0,0 +1,198 @@
...
+2. **Privacy**: With multiple DIDs, controlling which information is shared with whom is possible. This reduces the risk of profiling or tracking by third parties. Using different DIDs in different contexts makes it more difficult for observers to correlate all actions and interactions to a single identity.
+3. **Security**: If one DID is compromised, other DIDs and associated contexts remain secure. This isolates the risk and minimizes damage.
+4. **Flexibility and scalability**: Users can create new DIDs as their needs and contexts evolve without being limited by rigid identifiers.
+
+DIDs are managed by decentralized networks, notably blockchains. This means that no single central authority controls identity information, offering greater security and resilience against single points of failure. Users have total control over their DIDs. They can create, update, or delete their identifiers without the need for a central authority.
+
+The DID is a [W3C](https://www.w3.org/TR/did-core/) standard designed to be interoperable across different systems and networks, facilitating their use in diverse digital contexts.
+
+**ILLUSTRATION**
```
[the remark this answers is not in the document]
Comment (MdechampG) [reply docs/academy/decentralized-identity.mdx:83]: I will add the illustration later.

File: docs/academy/decentralized-identity.mdx (hunk 3 above)
[the remark this answers is not in the document]
Comment (MdechampG) [reply docs/academy/decentralized-identity.mdx]: Done ✅

File: docs/academy/decentralized-identity.mdx [hunk 5]
```diff
@@ -0,0 +1,198 @@
...
+
+### Verifiers
+
+A Verifier is an entity empowered to authenticate the legitimacy and veracity of a Verifiable Credential through a Verifiable Presentation submitted to it. The verifier can confirm that the information presented originates from the correct and authorized Issuer and that the Verifiable Credential has not been altered in any way. Generally, the verifier can also check that the Verifiable Credential has not expired or been revoked.
+
+**ILLUSTRATION**
+
+**Zero-knowledge proof** (ZKP) technology can be used to share proof of identity or authorization without revealing the complete identity of the holder. 
+
+Note that blockchain technologies generally support decentralized identity. Therefore, each data exchange between your wallet and an Issuer or Verifier may involve fees.
```
[the remark this answers is not in the document]
Comment (MdechampG) [reply docs/academy/decentralized-identity.mdx:39]: Thanks I integrated it

File: docs/academy/decentralized-identity.mdx [hunk 6]
```diff
@@ -0,0 +1,198 @@
...
+
+In the case of the declaration of a digital resource in the Dataverse, the Provider has three roles at the same time:
+- Issuer as he declares the metadata of its resource
+- Holder as he is the owner of the resource
+- Registrant as he declares the verifiable presentations in the OKP4 Protocol so that it can be used
+
+In the case of a declaration of reputation, the roles are handled by distinct entities:
+- The Issuer is a trusted authority
+- The Holder is the Provider subject of the reputation claim
+- The Registrant can be the Provider, the Issuer, or a third-party
```
[the remark this answers is not in the document]
Comment (MdechampG) [reply docs/academy/decentralized-identity.mdx]: The advantages are already discribed before the limitations.

**Review by ccamel — APPROVED** [review]
Perfect! :+1:

**Pull request merged**